Tests/SwiftStdlibTests/BoolExtensionsTests.swift

```diff
@@ -25,6 +25,18 @@ class BoolExtensionsTests: XCTestCase {
 		XCTAssertFalse(true.toggled)
 		XCTAssert(false.toggled)
 	}
+    
+    func testRandom() {
+        var yes = 0, no = 0
+        for _ in 1...10000 {
```

Comment:
It would be nice to not do this 10,000 times 😱 Just making note of it

Reply:
@SD10 I agree, 10 times is enough, will update it soon